ci/ansible/roles/pulp-smash/defaults/main.yml

```diff
@@ -9,3 +9,9 @@ pulp_smash_password: "admin"
 
 # If SSL certificate should be verified
 pulp_smash_verify: "false"
+
+# Pulp Server version
+# pulp_smash_version variable is optional, if you want to specify it then you
+# can do so by passing additional variables to ansible-playbook command or by
+# uncommenting the following line and editing as needed.
+# pulp_smash_version: 2.7
```

Comment:
I think it might be better to leave this uncommented, and don't recommend changing this value, since every other mechanism for defining vars overrides this one.

I'm not entirely clear on what this value is. How is it different from [pulp_version](https://github.com/pulp/pulp_packaging/blob/master/ci/ansible/roles/pulp/defaults/main.yaml#L10)?